tests/dataset_tests/CIFAR-10/Cifar10Test.cpp

```diff
@@ -61,7 +61,7 @@ TEST_F(Cifar10Test, DISABLED_trainBestNeuralNetwork)
         MaxPooling(2),
         FullyConnected(128),
         FullyConnected(10, activation::identity, Softmax())
-        },
+    },
```

Comment:
You can make this change in the first commit

Reply:
Yes but we don't care. (We're not at work)